### PR TITLE
fix 500 after login with email in prod

### PR DESCRIPTION
**Pull request by sumandari**
This PR to fix 500 error temporarily. It forces to redirect to /en/ after user logged in with username/email.

### CODE REVIEW

**Comment by lucernae** [comment]
@sumandari I've found some reference to set the default activated language in this page:

https://docs.djangoproject.com/en/2.2/topics/i18n/translation/#explicitly-setting-the-active-language

I propose that the root `'^$'` url uses a custom view function where it activates the current language in the cookie session. If the cookie doesn't have language set yet, then use default language specified in `settings.LANGUAGE_CODE`, which is probably `en`. After activating the language, the view function then returns a redirect to the reversed url of the index page (the one in the `base.urls`)

File: django_project/core/urls.py [hunk 1]
```diff
@@ -32,7 +33,11 @@ def handler500(request):
     }))
 
 
-urlpatterns = []
+urlpatterns = [
+    # force redirect to /en/
+    # to fix 500 error temporarily  after user logged in on production
+    url(r'^$', RedirectView.as_view(url='/en/')),
```
Comment (lucernae) [inline django_project/core/urls.py]: Can you try the following:

Replace this with:

```suggestion
    url(r'^$', index_view),
```

Where `index_view` defined as

```python
from django.utils import translation
from django.shortcuts import redirect
from django.urls import reverse

def index_view(request):
    user_language = translation.get_language() or settings.LANGUAGE_CODE or 'en'
    translation.activate(user_language)
    # I looked up the view name in base.urls
    return redirect(reverse('home'))
```

Comment (lucernae) [reply django_project/core/urls.py]: It might not work out of the box, but try and modify if possible.

**Review by lucernae — COMMENTED** [review]
Changes to set default language for the index view

File: django_project/core/urls.py (hunk 1 above)
Comment (sumandari) [reply django_project/core/urls.py]: Thanks @lucernae, I implement your code in the new commit. I gave it a try on my local machine and it worked as expected.

**Comment by sumandari** [comment]
The issue fixed by #1269

**Comment by sumandari** [comment]
reopen the PR, the error still exists

**Comment by lucernae** [comment]
LGTM. Try it in staging with DEBUG False so we can be sure.

**Pull request merged**